packages/@uppy/companion/src/server/controllers/url.js

```diff
@@ -93,8 +93,7 @@ const meta = async (req, res) => {
     return res.json(urlMeta)
   } catch (err) {
     logger.error(err, 'controller.url.meta.error', req.id)
-    // @todo send more meaningful error message and status code to client if possible
-    return res.status(err.status || 500).json({ message: 'failed to fetch URL metadata' })
+    return res.status(err.status || 500).json({ message: err.message || 'failed to fetch URL metadata' })
```

Comment:
not sure if we want to expose any error message to the client. maybe the current implementation is just fine? If we want to pass more specific error messages, I think we should classify and i18n them, so the user doesn't just see something like `Request to https://secret/url?key=abcdef" failed with status 404`

Reply:
Alright I can revert.